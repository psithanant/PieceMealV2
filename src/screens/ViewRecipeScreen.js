import React, { Component } from 'react';
import { Container } from 'native-base';
import RecipeView from '../components/Recipe/RecipeView';

export default class ViewRecipeScreen extends Component {
  render() {
    const params = this.props.navigation.state.params;
    return (
      <Container>
        <RecipeView recipe={params.recipe} />
      </Container>);
  }
}
