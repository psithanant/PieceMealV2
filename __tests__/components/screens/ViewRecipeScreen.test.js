import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ViewRecipeScreen from '../../../src/screens/ViewRecipeScreen';

describe('View Recipe Screen', () => {
  it('renders ViewRecipeScreen component', () => {
  const component = shallow(
    <ViewRecipeScreen
      navigation={{
          state: {
            params: {}
          }
        }}
    />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
