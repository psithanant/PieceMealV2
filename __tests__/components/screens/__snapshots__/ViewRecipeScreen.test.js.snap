// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Recipe Screen renders ViewRecipeScreen component 1`] = `
<Styled(Container)>
  <RecipeView />
</Styled(Container)>
`;
